### PR TITLE
Solves Session Expired Modal 

### LINKED ISSUE #5795: [UI] Session Expired: Use warning colors for modal
#### Current Behavior
This new Expired Session modal should be a warning not informational.

Untitled

#### Desired Behavior
Use Warning sign, not Info sign:

- [warning sign](https://github.com/meshery/meshery/files/9087585/alert.svg.zip)

 Use the following colors:

- Dark yellow: #F0A303
- Light yellow: #F0D053

#### Screenshots/Logs
Screen Shot 2022-07-11 at 4 09 07 PM

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by kamalbuilds**
**Description**
[UI] Session Expired: Use warning colors for modal 

This PR fixes #5795 

**Notes for Reviewers**
Dark yellow: #F0A303 is used for bg
Light yellow: #F0D053 is used for warning icon
and white for the text.

Attatching some ss 

image

image

Colors are used as per suggested by @leecalcote 
Info icon is successfully replaced with warning icon from material ui.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- Yes, I signed my commits.
 

Updated SS
After aligning vertically
image

### CODE REVIEW

File: provider-ui/pages/auth/login.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const AlertUnauthenticatedSession = ({ classes }) => {
       >
         <DialogTitle id="alert-dialog-title" className={classes.title}>
           <span className={classes.icon}>
-            <InfoIcon color="#2196f3"/>
+            <WarningIcon color="#F0D053"/>
```
Comment (Chadha93) [inline provider-ui/pages/auth/login.js:73]: Import this icon, otherwise, it won't work.

**Comment by legendarykamal** [comment]
Pls check the first commit -> that had major changes 
while second was for signing off 
thanks

**Review by Chadha93 — COMMENTED** [review]
@legendarykamal Please always provide screenshots or videos whenever a UI change is made. It would help the reviewers a lot.

File: provider-ui/pages/auth/login.js (hunk 1 above)
Comment (legendarykamal) [reply provider-ui/pages/auth/login.js:73]: ok would provide a ss :) 
Oh how did i miss i will import it right away

**Comment by legendarykamal** [comment]
SS of the new modal updated @Chadha93

File: provider-ui/pages/auth/login.js (hunk 1 above)
Comment (legendarykamal) [reply provider-ui/pages/auth/login.js:73]: It did work out as I had imported it earlier the reason for it to not not be present in commit was coz I was messing with the signing off , 
so I was trying to do little changes to commit and sign off :) but everything is good now .

**Comment by Chadha93** [comment]
Also, please sign-off your first commit in order to pass DCO.

**Comment by Chadha93** [comment]
Restarting failed UI e2e test.

**Comment by legendarykamal** [comment]
> @legendarykamal Thank you for adding ss, It looks your recent changes are introducing an unwanted bottom scrollbar, would you please look into it?

yes looking into it

**Comment by leecalcote** [comment]
updated screenshot looks good.

**Comment by legendarykamal** [comment]
@leecalcote done with the required changes , 

image

**Comment by leecalcote** [comment]
I hate to ask this, but will you use flexbox or relative positioning? This absolute positioning is too fragile.

**Comment by legendarykamal** [comment]
Sure @leecalcote i will 
The only reason I didn't do was it was earlier absolute positioned so 😅 .. 
anyways will do the reqd 👍

**Comment by legendarykamal** [comment]
I have preferred using relative positioning for the icon , hope looks good now

image

**Comment by leecalcote** [comment]
@legendarykamal is this an issue with the code change in this PR? https://github.com/meshery/meshery/runs/7526885118?check_suite_focus=true#step:6:49

**Comment by legendarykamal** [comment]
@leecalcote  no I have just made changes to the login.js as depicted in the file changes tab.
Those are not related to this PR

**Comment by leecalcote** [comment]
Yay, @legendarykamal! 🎈

**Comment by legendarykamal** [comment]
yayy now that looks amazingg 🥳🥳

**Pull request merged**